Code/training_team8.py

```diff
@@ -39,9 +40,10 @@
 data_loader_params = {
     'batch_size': BATCH_SIZE,
     'shuffle': True,  # why not.. *shrug*?
-    'num_workers': 12}
+    'num_workers': 16}
 
 training_loader, validation_loader = build_data_loaders(
+    base_label_transform,
```

Comment:
added an ability to pass in the label transformer so that the training file controls it. 
Should better facilitate copying training file for each model we're training.